transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/util/Util.java

```diff
@@ -681,42 +676,35 @@ private static void updateIndexes(Model model,
 
     public static void updateMavenDependencies(final List<Dependency> dependencies,
```

Comment:
I initially wrote my own utilities just to avoid the dependency since much of our code was previously used by a project that wasn't an Eclipse project.  Since it's all Eclipse projects now, I agree I can merge this code.